src/routes/routes.js

```diff
@@ -1,10 +1,11 @@
 import { renderLandingPage, trimUrl, deleteUrl } from "../middlewares/middlewares";
-
+import redirectUrl  from "../controllers/redirect-url"
 export default initRoutes = (app) => {
 	app.get('/', (req, res) => renderLandingPage(req, res));
 
 	app.post('/api/clip', (req, res) => trimUrl(req, res));
 
+	app.get('/api/clip/:id', (req, res) => redirectUrl(req, res));
```

Comment:
It can be
`app.get('/api/clip/:id', redirectUrl);`
and still work

Reply:
Ok thanks